lib/ddtrace/ext/distributed.rb

```diff
@@ -7,9 +7,9 @@ module DistributedTracing
       HTTP_HEADER_TRACE_ID = 'x-datadog-trace-id'.freeze
       HTTP_HEADER_PARENT_ID = 'x-datadog-parent-id'.freeze
       HTTP_HEADER_SAMPLING_PRIORITY = 'x-datadog-sampling-priority'.freeze
-      SAMPLING_PRIORITY_KEY = '_sampling_priority_v1'.freeze
+      TAG_SAMPLING_PRIORITY = '_sampling_priority_v1'.freeze
       HTTP_HEADER_ORIGIN = 'x-datadog-origin'.freeze
-      ORIGIN_KEY = '_dd.origin'.freeze
+      TAG_ORIGIN = '_dd.origin'.freeze
 
```

Comment:
Want to break this PR up (it is unreasonably big for review), but because these things touch so many files, it's very difficult to clean separate the code changes. e.g. there's a lot of lines where both `TAG_SAMPLING_PRIORITY` and `span.set_tag` changed; it's difficult to extract one from the other.

Reply:
Yeah, my suggestion would be after finishing the exploration/spike phase, then looking at the diff, and identifying what could reasonably go in separately; I realize when we're still moving and changing stuff it's very difficult to be peeling stuff away.